src/pycram/process_module.py

```diff
@@ -419,3 +419,9 @@ def close(self) -> Type[ProcessModule]:
         """
         raise NotImplementedError(
             f"There are no Process Modules for '{inspect.currentframe().f_code.co_name}' for robot '{self.robot_name}'")
+
+    def exit(self)->None:
```

Comment:
The exit method for Process modules is a nice idea. 
But the actual code to exit the connection to a robot should be directly in the Process module.
Please also add an exit method to the ProcessModule parent class and then call the currently loaded ProcessModule.exit method from this method

Reply:
isn't this what i did ? 
Would you explain more this part

Reply:
Yes what you did is correct, I was just confused about the structure of the ProcessModuleManager